### ISSUE TITLE
How do we organize this?

### ISSUE THREAD

**Issue by capr** [terralang/std#1]
Do we want a separate repo for discussion or do we use this one?

**Comment by aiverson** [comment]
I think it depends on what kind of discussion. I feel like we should have an initial meeting about organization in real time elsewhere rather than in these issues, document the decisions from that discussion into the README, and then use issues mostly for things like "should we add this to the stdlib" and "who is getting assigned to this feature" and "there is a bug in something"

**Comment by capr** [comment]
@aiverson ok, how about we have a chat on gitter?

btw @elliottslaughter, please make the org "read/write" by default so we can avoid the overhead of pull-requests.

**Comment by elliottslaughter** [comment]
@capr I believe I have done this, please let me know if there's anything else.

**Comment by aiverson** [comment]
No, pull requests are really important here. We should do code reviews on everything in the Standard Library. These things need to be relatively bug free and secure, so having code reviews is useful. We just need to have a good process for approving pull requests and doing code reviews, and a good architecture that supports that. I can add you both as collaborators here so that you have edit access and can approve PRs. We can rough out some basics without using a full review process, but I think we need to do code reviews and PRs at some point.

**Comment by capr** [comment]
If you wanna chat, I just made this: https://gitter.im/terralang-stdlib/community?utm_source=share-link&utm_medium=link&utm_campaign=share-link

**Comment by capr** [comment]
FYI: I deleted the other repo and added a welcome message to this one, so I propose to just use this one for both talking and code. Lemme add a first "real" talking point.

**Comment by Qix-** [comment]
Just chiming in: I think the realtime Gitter + this repository for discussion _and_ code is perfect. I vote to keep it as-is.

However, I'd say we should rename it to simply `terra/stdlib` or perhaps even `terra/std`. Prefixing the repository name with the organization name is redundant.

**Comment by capr** [comment]
**Chat moved to https://terralang.zulipchat.com**

**Comment by aiverson** [comment]
Is the zulip chat abandoned at this point?

**Comment by elliottslaughter** [comment]
I don't know about anyone else, but on my end I failed to set up email notifications... So not intentionally abandoned, no.
